src/NLog/GlobalDiagnosticsContext.cs

```diff
@@ -111,5 +131,17 @@ public static void Clear()
                 dict.Clear();
             }
         }
+
+        internal static string RenderContextObject ( object o )
```

Comment:
I like the solution, but I was confused by the methode name. Proposal: `ConvertToString`